apidoc/Titanium/Android/NotificationManager/NotificationManager.yml

```diff
@@ -48,6 +48,12 @@ methods:
     returns:
         type: Titanium.Android.NotificationChannel
 
+  - name: areNotificationsEnabled
+    summary: Returns whether showing notifications is enabled for the application.
+    returns:
+        type: Boolean
+    since: "7.1.0"
```

Comment:
We need more parity for push notifications. Why not use [remoteNotificationsEnabled](http://docs.appcelerator.com/platform/latest/#!/api/Titanium.Network-property-remoteNotificationsEnabled)?

Reply:
That option will also block local notifications, so that wording would not describe it's full effect.